### ISSUE TITLE
Update menu suggest to update to wrong version

### ISSUE THREAD

**Issue by felipepessoto** [aspnet/LibraryManager#633]
### Describe the bug
The same issue we had here: https://github.com/aspnet/LibraryManager/issues/359, but it is happening in the update menu

### To Reproduce
Use the update menu to update the Prism library.

### Expected behavior
The latest version is 1.24.1. But it suggest to instal v9000

### Screenshots
![image](https://user-images.githubusercontent.com/1336227/124657032-5ba65000-de78-11eb-98b3-9d7dc976da8c.png)

**Comment by jimmylewis** [comment]
This is using cdnjs, right?  It looks like they believe there is a version 9000 of that package: https://cdnjs.com/libraries/prism/9000.0.1.  Can you try following up with them directly on https://github.com/cdnjs/cdnjs/issues to see where they're getting it from?  We just reflect the data that they provide.

**Comment by felipepessoto** [comment]
Yes, there is a version 9000, but the correct way to get the latest version is looking at Version property, more details in the issue I mentioned above
